app/src/main/res/values/dimens.xml

```diff
@@ -51,7 +51,7 @@
     <dimen name="text_extra_small">12sp</dimen>
     <dimen name="text_small">16sp</dimen>
     <dimen name="text_normal">20sp</dimen>
-    <dimen name="text_large">28sp</dimen>
+    <dimen name="text_large">25sp</dimen>
```

Comment:
I'm sorry, I just checked that they have been used in intro1.xml and a couple of other places. I shall create another entry in dimens.xml and restore it back to 28sp? 
Are the other things fine, so that I can make the next commit without any more bugs?